ParabolInc/parabol
# feat: allow disable anonymity for retrospective meeting reflections

Fixes: #7032

<img width="877" alt="image" src="https://user-images.githubusercontent.com/466991/185436380-b328bbd5-3650-4013-941f-50f4c15d092e.png">

<img width="648" alt="image" src="https://user-images.githubusercontent.com/466991/185436538-80adf47f-db54-4367-94f8-3e7ca2c30944.png">



Demo:

https://www.loom.com/share/5f10ba990517438ca4b6048342bf2c65

How to test:

- Create a retro meeting, see that reflections are anonymous by default
- In new meeting dialog remove checkbox from Anonymous reflections option, create a retro meeting, see reflections now have name attached
- Try to create various reflections and interact with them
- Refresh page, go to meeting dialog, see latest settings are saved
- Smoke test that other meetings are created normally and icebreaker checkbox works too
